recipes/google-cloud-cpp-compute/meta.yaml

```diff
@@ -0,0 +1,128 @@
+{% set version = "2.17.0" %}
+
+package:
+  name: google-cloud-cpp-compute
+  version: {{ version }}
+
+source:
+  url: https://github.com/googleapis/google-cloud-cpp/archive/v{{ version }}.tar.gz
+  sha256: 8cb87ec2947e691a7f8631877e78453bcfda51b3d3cd4940794a376741888d37
+  patches:
+    # This is an upstream fix to allow sharding google/cloud/oauth2
+    - patches/0001-Backport-12911-from-upstream.patch
+
+build:
+  number: 0
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - libgrpc
+    - libprotobuf
+    - libgoogle-cloud-devel =={{ version }}
+    - zlib # Needed by gRPC
+  host:
+    - libabseil
+    - libcurl
+    - libgrpc
+    - libprotobuf
+    - openssl
+    - nlohmann_json
+    - zlib # Needed by gRPC
+
+outputs:
+  - name: libgoogle-cloud-compute
+    script: install-libgoogle-cloud.sh   # [unix]
+    script: install-libgoogle-cloud.bat  # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+        - libgoogle-cloud-devel =={{ version }}
+      host:
+        - libgoogle-cloud-devel =={{ version }}
+        - libabseil
+        - libprotobuf
+        - openssl
+        - libcxx  # [osx]
+      run:
+        - libgoogle-cloud =={{ version }}
+        - libabseil
+        - libprotobuf
+        - openssl
+        - libcxx  # [osx]
```

Comment:
All this should be covered by run-exports

Reply:
Fixed.